scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -62,6 +88,32 @@ trait AndroidAppModule extends JavaModule {
    */
   def androidReleaseKeyPath: T[PathRef] = Task.Source(millSourcePath / androidReleaseKeyName())
 
+  /**
+   * Specifies the file format of lint report. Available file formats are "html", "xml", "txt" and "sarif".
+   */
+  def androidLintReportFmt: T[AndroidLintReportFormat.Value] = Task { AndroidLintReportFormat.Html }
```

Comment:
you can relatively easy support generating report in the multiple formats at the same time if this configuration option will be `T[Set[AndroidLintReportFormat.Value]]`

In case if it is helpful: tool source code is here https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:lint/cli/src/main/java/com/android/tools/lint/